openedx/core/lib/request_utils.py

```diff
@@ -1,27 +1,17 @@
 """ Utility functions related to HTTP requests """
-
-import logging
 import re
-
 import crum
+from urllib.parse import urlparse
+
 from django.conf import settings
 from django.test.client import RequestFactory
-from django.utils.deprecation import MiddlewareMixin
-from edx_django_utils.monitoring import set_custom_attribute
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
-from six.moves.urllib.parse import urlparse
 
-from edx_toggles.toggles import LegacyWaffleFlag, LegacyWaffleFlagNamespace
-from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
```

Comment:
Not sure if there are other issues, but looks like this `configuration_helpers` import needs to be restored.